css/styles.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+
+  line-height: 1.7;
+}
+
+body {
+  background-color: white;
+}
+
+@media (max-width:1000px) {
+  body {
+      background-color: rgb(120, 179, 120);
+  }
+
+}
+
+#navbar {
+font-family:Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
Please try to stick to the template you used. The template uses Poppins as a font-family. The template uses Poppins as a font-family for most of the texts in the document. Therefore, put your font-family inside boy CSS selector.

Reply:
@amare1990 kindly try to include an explanation "why" for your feedback, to help the student understand why you requesting the changes